src/bin/builders.rs

```diff
@@ -0,0 +1,43 @@
+//! # Builders Sample
+//!
+//! This sample demonstrates how to create a widget using the builders.
+//! These allow to set construct-only properties and other construct
+//! properties when creating the widget.
+
+extern crate gio;
+extern crate gtk;
+
+use gio::prelude::*;
+use gtk::prelude::*;
+
+use std::env::args;
+
+fn build_ui(application: &gtk::Application) {
+    let window = gtk::ApplicationWindow::new(application);
+
+    window.set_title("First GTK+ Program");
+    window.set_border_width(10);
+    window.set_position(gtk::WindowPosition::Center);
+    window.set_default_size(350, 70);
```

Comment:
Would be nice to also have a builder for the above things :)

Reply:
There's no builder generated for `ApplicationWindow`.
To be able to do this, that would require the builder to generate methods for parent properties. I'll see if I can do that.

Reply:
... which happened AFAIU now? :)